tests/test_autoscaling_lib.py

```diff
@@ -922,10 +923,11 @@ def test_set_spot_fleet_request_capacity():
         mock.patch('paasta_tools.autoscaling_lib.AWS_SPOT_MODIFY_TIMEOUT', autospec=True)
```

Comment:
Can you rename this test to be something like "set_sfr_times_out" or something? to make it more obvious what this is testing